cache/cache_helpers.py

```diff
@@ -68,3 +68,14 @@ def _increment_stat(self, stat: Statistic, char: str):
             case "Watcher":
                 stat.watcher_count += 1
         stat.all_character_count += 1
+
+class RunLinkedListNode:
```

Comment:
this is just a prototype to replace the `matched` attribute on `RunParser`. But I don't actually want to do it yet since this was only about clicking the key and node info showing up lol.

If you're ok with this pattern though I'll replace the `matched` attribute where necessary